actor_library/src/token/erc20.rs

```diff
@@ -0,0 +1,560 @@
+use crate::utils::{map, resolve_address};
+use anyhow::anyhow;
+use cid::Cid;
+use fvm_ipld_blockstore::Blockstore;
+use fvm_ipld_hamt::BytesKey;
+use fvm_shared::address::Address;
+use fvm_shared::event::{Entry, Flags};
+use fvm_shared::IPLD_RAW;
+use num_bigint::BigUint;
```

Comment:
We should use BigInt/BigUint from fvm_shared library
```suggestion
use fvm_shared::bigint::BigUint
```

Reply:
Fixed [here](https://github.com/Ethernal-Tech/filecoin-actor-library/pull/9/commits/8f4a670cb13b535b176f0710e5c46cab9dab6f7e#diff-a4cbe2dcb4017985d5aa71f1463dfd9c74f2a66855589672440368fa96a2adb5).